tests/tts/test_tts.py

```diff
@@ -189,3 +192,95 @@ def test_waveglow_training(self):
         optimizer.train(
             [loss_t], callbacks=[callback], optimizer="sgd", optimization_params={"num_epochs": 10, "lr": 0.0003},
         )
+
+    def test_fastspeech(self):
```

Comment:
Let's sit on this change until after we merge #430

Reply:
Can you add a test label that is compatible with #454

Reply:
Done.

Reply:
Unclassified is not an option.
I would mark this as integration or system. Depending on what you want to test. Since you made a unittest, I assume it's more of a integration test, in which case you should only run it for 1 or 2 steps.

If you want to change it to a system test, please drop num_epochs to be between 1-3 and check that loss or other metric acts as expected.

Reply:
marked as integration